db/re/item_db_usable.yml

```diff
@@ -61100,7 +61100,7 @@ Body:
     Type: Usable
     Weight: 10
     Script: |
-      addhomintimacy 100; // !TODO: Confirm increase amount
+      addhomintimacy 250;
```

Comment:
on iRo Wiki say 50(this need more than 40 uses to first up level of intimacy), when seeing videos from kro, upcost it's around 15-20-25-30(...) pills to increase intimacy, in my test its close to 250 intimacy per use

Reply:
Be careful following iRO. Their skill content doesn't always match kRO.

Reply:
I followed the only video about the pill update in kro that I found, but I think this value is higher, apparently I misunderstood what he says in the video, he says that around 15 pills are needed, I thought it was per grade of loyalty, but in the video he uses 10x pills and his homunculus' loyalty increases when he consumes the 3rd and 10th pill, and he used 7 pills for loyalty to go from neutral to cordial, according to my calculations this comes to around 7100+ of intimacy by use, is that correct?

video reference -> https://www.youtube.com/watch?v=DUJzfcSPnRI

Reply:
We have confirmed the item script should be:
```
addhomintimacy rand(10,100);
```